udfs/community/test_cases.js

```diff
@@ -3752,3 +3752,14 @@ generate_udf_test("cw_xml_extract", [
     expected_output: `"xy"`,
   },
 ]);
+
+generate_udf_test("gunzip", [
+  {
+    inputs: [`FROM_BASE64("H4sIACYJJWgAA8tIzcnJVyjPL8pJ4QIALTsIrwwAAAA=")`],
+    expected_output: `CAST("hello world" AS BYTES)||FROM_BASE64("Cg==")`,
```

Comment:
Thanks, I wasn't sure what I should put here. I left the value as it appears in the BQ console, but it makes sense that we can use SQL here as well, I even used that in for the `NULL` case!

That said, I just realized that I did `echo "hello world" | gzip | base64` which is why there's a newline. We should probably just put the value without a newline: `H4sIABDwJWgAA8tIzcnJVyjPL8pJAQCFEUoNCwAAAA==` instead (`echo -n "hello world"`). So that the expected output is just `CAST("hello world" AS BYTES)`.

Reply:
I agree, please update to remove the newline from the example and unit test. Thanks!

Reply:
Done in https://github.com/GoogleCloudPlatform/bigquery-utils/pull/493/commits/a8a6b9de3dde81e3b1a9ce0545c0ee7a9e3b4d03

Thanks for all the fixes!